app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt

```diff
@@ -591,6 +592,32 @@ class PasswordStore : AppCompatActivity(R.layout.activity_pwdstore) {
         startActivityForResult(intent, REQUEST_CODE_SELECT_FOLDER)
     }
 
+    fun renameCategory(values: Stack<PasswordItem>) {
+        if (values.isNotEmpty()) {
+            val value = values.pop()
+            if (value.type == PasswordItem.TYPE_CATEGORY) {
```

Comment:
Rather than check here we should probably reinstate the visibility code from [here](https://github.com/android-password-store/Android-Password-Store/commit/d8231e112afb501c43041a6f839ab8285f400f77#diff-8aba73d039d20af1e4e0c11c80727919L158-L161)